Source/WebCore/en.lproj/Localizable.strings

```diff
@@ -1096,6 +1102,9 @@
 /* Title for Open in External Application Link action button */
 "Open in “%@”" = "Open in “%@”";
 
+/* Open this 3D model? */
```

Comment:
We should probably preserve the original comment, which was `Open 3D object in a new window?`.